test/01_basic/cibuildwheel_basic_test.py

```diff
@@ -1,16 +1,20 @@
 import os
-import utils
+import sys 
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+
+from shared import utils
 
 
 project_dir = os.path.dirname(__file__)
 
-def test():
+def test(tmp_path):
     # build the wheels
-    utils.cibuildwheel_run(project_dir)
+    utils.cibuildwheel_run(project_dir, output_dir=tmp_path)
 
     # check that the expected wheels are produced
     expected_wheels = utils.expected_wheels('spam', '0.1.0')
-    actual_wheels = os.listdir('wheelhouse')
+    actual_wheels = [x.name for x in tmp_path.iterdir()]
```

Comment:
why not keep `os.listdir ? I find this less readable at first glance.

Reply:
because of python 2.7. 
For me this is more elegant than convert tmp_path to string), 

```
>>> import os 
>>> from pathlib2 import Path
>>> p = Path(".")
>>> list(p.iterdir())
[PosixPath('setup.cfg'), PosixPath('bin'), PosixPath('.travis.yml'), PosixPath('cibuildwheel'), PosixPath('azure-pipelines.yml'), PosixPath('.git'), PosixPath('.vscode'), PosixPath('pytest.ini'), PosixPath('setup.py'), PosixPath('docs'), PosixPath('.circleci'), PosixPath('.readthedocs.yml'), PosixPath('.python-version'), PosixPath('requirements-dev.txt'), PosixPath('mkdocs.yml'), PosixPath('LICENSE'), PosixPath('unit_test'), PosixPath('README.md'), PosixPath('test'), PosixPath('CODE_OF_CONDUCT.md'), PosixPath('.gitignore'), PosixPath('.pytest_cache'), PosixPath('appveyor.yml'), PosixPath('cibuildwheel.egg-info')]
>>> os.listdir(p)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: coercing to Unicode: need string or buffer, PosixPath found
>>> os.listdir(str(p))
['setup.cfg', 'bin', '.travis.yml', 'cibuildwheel', 'azure-pipelines.yml', '.git', '.vscode', 'pytest.ini', 'setup.py', 'docs', '.circleci', '.readthedocs.yml', '.python-version', 'requirements-dev.txt', 'mkdocs.yml', 'LICENSE', 'unit_test', 'README.md', 'test', 'CODE_OF_CONDUCT.md', '.gitignore', '.pytest_cache', 'appveyor.yml', 'cibuildwheel.egg-info']
```

Reply:
It is indeed more elegant to get rid of the string conversion, but I was talking about readability.
I still think `os.listdir(str(tmp_path))` is more readable than `[x.name for x in tmp_path.iterdir()]`.